Userland/Libraries/LibPDF/Fonts/Type0Font.h

```diff
@@ -19,6 +22,9 @@ struct CIDSystemInfo {
 
 class Type0Font : public PDFFont {
 public:
+    Type0Font() noexcept;
```

Comment:
Why `noexcept`? Just wondering, I've rarely seen this outside of AK and the Kernel.

Reply:
No good reason. IIRC i forgot the declaration at first and the compiler said "definition doesn't match declaration, add noexcept* (because the implicitly ctor has it). I can remove it if you want.

Reply:
Removed it.